### DISCUSSION TITLE
全套硬件的采购价格是多少？

### DISCUSSION THREAD

**Discussion by 17837697** [peng-zhihui/ElectronBot#58] [Q&A]
全套硬件的采购价格是多少？
有套装卖吗？

**Comment by peng-zhihui** [comment]
我不卖东西

**Reply by 17837697** [reply]
感谢大佬翻牌。
非专业人员（上次焊的电路板还是20年前上大学电子实习时的收音机），又对这个项目的产品感兴趣，想尝试复刻，所以想看看有没有像之前的那些项目一样卖套件或成品的。。。。。。
ps：首富家用“ElectronBot”搜索，发现现在已经有了，目前售价在500-1000左右。

**Reply by LebranceBW** [reply]
淘宝是真的快啊

**Reply by sc543753481** [reply]
> 淘宝是真的快啊

确实，哈哈，那家店还有别的复刻的

**Comment by track-tech** [comment]
套件可以问问群友看有没有开车的

**Comment by jinyi7016** [comment]
感觉主要是3D打印的价格高一些，其他的不是很高。

**Comment by DylanZin** [comment]
可以一起组团买零件，加个群交流 983344575
